cmd/ipfs-cluster-follow/commands.go

```diff
@@ -486,6 +486,43 @@ func listCmd(c *cli.Context) error {
 	return nil
 }
 
+// Stop
+func stopCmd(c *cli.Context) error {
+	clusterName := c.String(clusterNameFlag)
+
+	absPath, _, identityPath := buildPaths(c, clusterName)
+	if !isInitialized(absPath) {
+		printNotInitialized(clusterName)
+		return cli.Exit("", 1)
+	}
+
+	// if unpin flag is set, check if the ipfs-cluster-follow is running and unpin everything
+	if c.Bool("unpin") {
+		if _, err := os.Stat(filepath.Join(absPath, "api-socket")); err == nil {
```

Comment:
I also think it would be worth to let getClient return the client `endp`, and have an `localEndpoint(endp)` function that extracts the path of the unix socket from the multiaddress, or errors when not using a unix socket (in which case we cannot kill the process).